lnurl.py

```diff
@@ -92,7 +92,7 @@ async def api_lnurl_callback(
         wallet_id=link.wallet,
         amount=int(amount / 1000),
         memo=link.description,
-        unhashed_description=unhashed_description,
+        unhashed_description=None if get_funding_source().__class__.__name__ == "OpenNodeWallet" else unhashed_description,
```

Comment:
this is an ugly hack and a layering violation - maybe there is a better way how to fix the issue?

Reply:
what about adding a member for each wallet class like is_deschash_required and checking it here ?